### ISSUE TITLE
install py3d from build

### ISSUE THREAD

**Issue by takanokage** [IntelVCL/Open3D#277]
This is about installing py3d module when building from sources.
This is different from #217.

Currently the Open3D based python code has to start with something like this:

```
import sys, os
sys.path.append("../..")
from py3d import *
```
where the location to py3d is explicitly specified.

Instead we could simply use the following:

```
import py3d
```
if the py3d module gets installed.

**Comment by takanokage** [comment]
True for the case you're describing.
However I'm talking about what we can do when building from sources. I believe I have a solution, will initiate a PR hopefully later today.

**Comment by takanokage** [comment]
I have tested my solution for this on Ubuntu and Mac and it works. I can import py3d from anywhere on the disk.
I'll hold onto the PR until I figure out how to do the same on Windows.

**Comment by syncle** [comment]
Addressed in #286. Thanks @takanokage!

**Comment by takanokage** [comment]
thank you!
